### DISCUSSION TITLE
BHoM node to grab EC3 data

### DISCUSSION THREAD

**Discussion by BIMnz** [BHoM/BHoM#1459] [Q&A]
Hi, there

I'm trying to use Dynamo to do the Embodied carbon measure. 
Basically I'm following this video to replicate the scripts:
https://www.autodesk.com/autodesk-university/class/Using-Revit-and-Dynamo-Assess-Embodied-Carbon-2020#video

I used the CreateData, then select ---  LifeCycleAccessment  ----  EC3. Then I chose one of it. 
Then I used the "Explode" note to see what's inside of it. It shows nothing. 

Did I miss a step? How does the EC3 data linked to BHoM?
I tried the "Source link" for this node, it linked me to EC3's home page:  https://www.buildingtransparency.org/
![2022-12-02 15_20_38-Dynamo](https://user-images.githubusercontent.com/119640469/205200197-54ba7199-b951-4cdb-864e-d542d47a79d9.png)
![2022-12-02 15_21_38-Dynamo](https://user-images.githubusercontent.com/119640469/205200313-1b0e0dfb-b339-479a-b80d-1084e4c6892a.png)

BTW, what can I do to add a dataset? I need to add the New Zealand version of dataset to do the calculation. 
CO₂NSTRUCT       
https://www.branz.co.nz/environment-zero-carbon-research/framework/branz-co2nstruct/

Sorry, I'm really new to this. it might be a dumb question. 

I really appreciate any help you can provide.
Feng

**Comment by FraserGreenroyd** [comment]
@IsakNaslundBh @michaelhoehn can you assist?
